### DISCUSSION TITLE
Is this extension still useful with Chrome 110 which has a similar feature?

### DISCUSSION THREAD

**Discussion by BoD** [gioxx/MarvellousSuspender#203] [General]
Hi! Sorry for this question but I am wondering if this extension is still useful since the latest version of Chrome now does this automatically ([source](https://arstechnica.com/gadgets/2023/02/chrome-110-will-automatically-discard-background-tabs-heres-how-to-stop-it/)).

**Comment by ossilator** [comment]
chrome's discard feature isn't exactly new (like, 7+ years old), so not sure why this pops up (again) now.

anyway, TMS still has some advantages, in particular allowing to suspend the current foreground tab (which discard as currently implemented is fundamentally incompatible with, as being the foreground tab is specifically what re-activates it).

TMS also offers some more control over which tabs are (un-)suspended, though a dedicated extension like https://github.com/rkodey/the-great-er-discarder-er will do that, too.
